fixtures/american-dj/encore-lp12z-ip.json

```
@@ -0,0 +1,580 @@
+          "parameter": 2500,
+          "comment": "Hz"
+        },
+        {
+          "dmxRange": [136, 150],
+          "type": "Maintenance",
+          "parameter": 4000,
+          "comment": "Hz"
+        },
+        {
+          "dmxRange": [151, 165],
+          "type": "Maintenance",
+          "parameter": 5000,
+          "comment": "Hz"
+        },
+        {
+          "dmxRange": [166, 180],
+          "type": "Maintenance",
+          "parameter": 10000
+        },
+        {
+          "dmxRange": [181, 195],
+          "type": "Maintenance",
+          "parameter": 15000,
+          "comment": "Hz"
+        },
+        {
+          "dmxRange": [196, 210],
+          "type": "Maintenance",
+          "parameter": 20000,
+          "comment": "Hz"
+        },
+        {
+          "dmxRange": [211, 225],
+          "type": "Maintenance",
+          "parameter": 25000,
+          "comment": "Hz"
+        },
+        {
+          "dmxRange": [226, 229],
+          "type": "Maintenance",
+          "comment": "Enable Zoom Mode 1"
+        },
+        {
+          "dmxRange": [230, 233],
+          "type": "Maintenance",
+          "hold": "3s",
+          "comment": "Enable Zoom Mode 2"
+        },
+        {
+          "dmxRange": [234, 238],
+          "type": "Maintenance",
+          "comment": "Zoom Reset"
+        },
+        {
+          "dmxRange": [239, 255],
+          "type": "NoFunction"
+        }
+      ]
+    },
+    "Auto Program": {
+      "capabilities": [
+        {
+          "dmxRange": [0, 31],
+          "type": "Maintenance",
+          "parameter": "off"
+        },
+        {
+          "dmxRange": [32, 63],
+          "type": "Maintenance",
+          "comment": "Auto Program 1"
+        },
+        {
+          "dmxRange": [64, 95],
+          "type": "Maintenance",
+          "comment": "Auto Program 2"
+        },
+        {
+          "dmxRange": [96, 127],
+          "type": "Maintenance",
+          "comment": "Auto Program 3"
+        },
+        {
+          "dmxRange": [128, 159],
+          "type": "Maintenance",
+          "comment": "Auto Program 4"
+        },
+        {
+          "dmxRange": [160, 191],
+          "type": "Maintenance",
+          "comment": "Auto Program 5"
+        },
+        {
+          "dmxRange": [192, 223],
+          "type": "Maintenance",
+          "comment": "Auto Program 6"
+        },
+        {
+          "dmxRange": [224, 255],
+          "type": "Maintenance",
+          "comment": "Auto Program 7"
+        }
+      ]
+    },
+    "Auto Program Speed": {
+      "capability": {
+        "type": "Speed",
+        "speedStart": "slow",
+        "speedEnd": "fast"
+      }
+    },
+    "Auto Program Fade": {
+      "capability": {
+        "type": "EffectParameter",
+        "parameterStart": "low",
+        "parameterEnd": "high",
+        "comment": "minimum to maximum fade"
+      }
+    }
+  },
+  "modes": [
+    {
+      "name": "6ch",
```

```suggestion
      "name": "6-channel",
      "shortName": "6ch",
```

Please also change this for the other channels.